### PR TITLE
Resolve #1151 -- Minion Waypoint Fix

### LINKED ISSUE #1151: Chaos Minions Waypointing to Midlane
Chaos minions are wrongly getting assigned to midlane because the function GetLaneID() checks for "_C" in the file name, which is present in all chaos minions in the form of [_C]haos.

Fix. Probably create a separate function dedicated to only spawnBarracks, since it's they are the only files that have this issue

### PR DESCRIPTION
**Pull request by cabeca1143**
Fixed Bot Minions going to mid lane.
Resolve #1151

### CODE REVIEW

File: GameServerLib/Config.cs [hunk 1]
```diff
@@ -256,6 +256,26 @@ public LaneID GetLaneID()
                 return laneId;
             }
 
+            public LaneID GetSpawBarrackLaneID()
```
Comment (danil179) [inline GameServerLib/Config.cs]: TYPO: Should be `Spawn` instead of `spaw`.

**Review by danil179 — CHANGES_REQUESTED** [review]
There is a typo. Will merge without testing as it seems correct to me.

File: GameServerLib/Config.cs (hunk 1 above)
Comment (cabeca1143) [reply GameServerLib/Config.cs]: Resolved

**Review by danil179 — APPROVED** [review]
Approved. Hopefully it works.

**Pull request merged**